docs/layouts/partials/header/upper-widget.html

```diff
@@ -1,2 +1,7 @@
-<!-- Platform.sh User Widget -->
-<div id="user-widget"></div>
+<!-- Links to log in or start a trial with Platform.sh -->
+<div class="console_links">
+  <div class="console_links__wrapper">
+    <a class="console_links__login" href="https://console.platform.sh/" rel="noopener noreferrer">Log in</a>
+    <a class="console_links__free-trial" href="https://auth.api.platform.sh/register" rel="noopener noreferrer">Free trial</a>
+  </div>
+</div>
```

Comment:
It might not be a bad idea to add a line here.

Reply:
Is there a process you think this affects? I added one, but it's definitely not the only file we have without an empty line at the end and we don't have code styling guidelines.

Reply:
It's not a big deal, but just to ensure that there aren't a picky POSIX system that could complain.

Reply:
Given the >100 files like this we have, I don't think such a system is an issue.